src/builder/Build.kt

```diff
@@ -20,6 +20,29 @@ class Build() {
     val totalHp get() = lane.getTotalHp(currentLevel)
     val totalDps get() = lane.getTotalDps(currentLevel)
 
+    fun save(ds:DSFactory.DataStream) {
+        if( legion!=null ) {
+            val index = LegionData.legions.indexOf(legion!!)

```

Comment:
Not better to use the legion id here? I'm unsure about this.

Reply:
I am unsure too. Using the string id for legion (same thing with units) will increase the permalink size by huge factor (~15 chars vs 2 byte). I'll test the string version with a big build and we'll see.